R/cache.R

```diff
@@ -7,9 +7,8 @@
 #' @return 0 for success, 1 for failure, invisibly.
 #' @export
 clear_cache <- function(file = NULL, path = NULL) {
```

Comment:
Previously `*_cache()` messed up the test case for lint caching, because it loaded the exclusions from the package level `.lintr`.
Before fixing the relative path issue, those exclusions had no effect. Now with the correct relative paths, the bad test files are excluded from linting whenever `read_settings()` is called from somewhere within lintr, including `tests/testthat`.